### PR TITLE
Fix /applications/using_module/{module_name}/{module_version}/{version_type} does not return anymore platforms that used to contain the target module - close #765

### LINKED ISSUE #765: /applications/using_module/{module_name}/{module_version}/{version_type} returns platforms that used to contain the target module
C'est parce qu'on ignore pas les _deployed modules_ avec un ID nul dans `MongoPlatformRepository.findPlatformsUsingModule`...

### PR DESCRIPTION
**Pull request by Lucas-C**

### CODE REVIEW

File: core/domain/src/main/java/org/hesperides/core/domain/platforms/queries/views/ModulePlatformView.java [hunk 1]
```diff
@@ -6,4 +6,7 @@
 public class ModulePlatformView {
     String applicationName;
     String platformName;
+    public String toShortString() {
```
Comment (thomaslhostis) [inline core/domain/src/main/java/org/hesperides/core/domain/platforms/queries/views/ModulePlatformView.java]: Pourquoi pas `toString` ? :)

File: core/infrastructure/src/main/java/org/hesperides/core/infrastructure/mongo/platforms/MongoPlatformRepository.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
     @Query(value = "{ 'key.applicationName' : ?0 }", fields = "{ 'deployedModules' : 0 }")
     List<PlatformDocument> findPlatformsForApplicationAndExcludeModules(String applicationName);
 
-    @Query(value = "{ 'deployedModules': { $elemMatch: { 'name' : ?0, 'version' : ?1, 'isWorkingCopy' : ?2 }}}", fields = "{ 'key' : 1 }")
+    @Query(value = "{ 'deployedModules': { $elemMatch: { 'name' : ?0, 'version' : ?1, 'isWorkingCopy' : ?2, 'id' : { $gt: 0 } }}}", fields = "{ 'key' : 1 }")
```
Comment (thomaslhostis) [inline core/infrastructure/src/main/java/org/hesperides/core/infrastructure/mongo/platforms/MongoPlatformRepository.java]: 👍

File: tests/bdd/src/test/resources/platforms/get-platforms-using-module.feature [hunk 3]
```diff
@@ -19,4 +19,12 @@ Feature: Get platforms using module
     And an existing module named "module-b" with version "1"
     And an existing platform named "P3" with this module
     When I get the platforms using this module
-    Then a single platform is retrieved
+    Then 1 platform is retrieved
+
+  #issue-765
+  Scenario: get platforms using a module that was removed from a platform
```
Comment (thomaslhostis) [inline tests/bdd/src/test/resources/platforms/get-platforms-using-module.feature]: 👍

File: core/domain/src/main/java/org/hesperides/core/domain/platforms/queries/views/ModulePlatformView.java (hunk 1 above)
Comment (Lucas-C) [reply core/domain/src/main/java/org/hesperides/core/domain/platforms/queries/views/ModulePlatformView.java]: Ok, c'est modifié

**Pull request merged**